usr/src/uts/common/os/log_sysevent.c

```diff
@@ -641,6 +642,10 @@ sysevent_add_attr(sysevent_attr_list_t **ev_attr_list, char *name,
 		error = nvlist_add_hrtime(*ev_attr_list, name,
 		    se_value->value.sv_time);
 		break;
+	case SE_DATA_TYPE_NVLIST:
```

Comment:
Yes, nested nvlists are perfectly valid nvlists.  But on FreeBSD, the kernelland->userland interface for sysevents doesn't use nvlists; it uses a flat array of KV pairs, formatted as a string.  So it would be nice if OpenZFS doesn't add any nested nvlists to ZFS sysevents.  If it does, then it will have to be changed for FreeBSD.